client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/ILMDocumentationIT.java

```diff
@@ -631,16 +631,22 @@ public void testRetryPolicy() throws Exception {
         // end::ilm-retry-lifecycle-policy-request
 
 
-        // tag::ilm-retry-lifecycle-policy-execute
-        AcknowledgedResponse response = client.indexLifecycle()
-            .retryLifecyclePolicy(request, RequestOptions.DEFAULT);
-        // end::ilm-retry-lifecycle-policy-execute
+        try {
+            // tag::ilm-retry-lifecycle-policy-execute
+            AcknowledgedResponse response = client.indexLifecycle()
+                .retryLifecyclePolicy(request, RequestOptions.DEFAULT);
+            // end::ilm-retry-lifecycle-policy-execute
 
-        // tag::ilm-retry-lifecycle-policy-response
-        boolean acknowledged = response.isAcknowledged(); // <1>
-        // end::ilm-retry-lifecycle-policy-response
+            // tag::ilm-retry-lifecycle-policy-response
+            boolean acknowledged = response.isAcknowledged(); // <1>
+            // end::ilm-retry-lifecycle-policy-response
 
-        assertTrue(acknowledged);
+            assertTrue(acknowledged);
+        } catch (ElasticsearchException e) {
+            // the retry API might fail as the shrink action steps are retryable (so if the retry API reaches ES when ILM is retrying the
+            // failed `shrink` step, the retry API will fail)
+            // as this code is more for illustrative purposes in the docs we'll allow it
```

Comment:
Is there any way we can check the message or something else here? I don't want us to accidentally miss a bug in `retryLifecyclePolicy` that is unrelated because we swallowed the exception

Reply:
Great point. Pushed a fix to assert the message. Thanks Lee